### ISSUE TITLE
User input malformed when pressing Submit

### ISSUE THREAD

**Issue by TheArcturian** [jseidl/hass-magic_areas#111]
I get the error "User input malformed" in red text when I pull up a area and just press "Submit" without changing any settings. Same if I try to change something as well.

It is like this in only some of the areas and not all. I don't know which setting is causing the problem which makes it impossible to do any changes in some of the areas.

**Comment by KTibow** [comment]
Anything of logs? Screenshot of config?

**Comment by TheArcturian** [comment]
I was wondering if it was a known issue, so I did not include screenshot or logs. Now that you asked for it, I found the issue.

I had selected one of the lights in the area as "Exclude entities from being analyzed", but I had already saved the same light to be used as "Sleep lights".

So I noticed it was impossible to deselect the sleep light as it wasn't in the list. I guess this is a bug as one particular light should be able to be used as a sleep light only. So I presume excluding the light from being analyzed shouldn't remove it in the sleep light list.

Anyways, thanks for asking the questions. :-)
